index.bs

```diff
@@ -386,12 +388,61 @@ To <dfn abstract-op>initialize a global object's embedder policy from a response
                 item's [=embedder policy/report only reporting endpoint=] is non-null, then
                 |policy|'s [=embedder policy/report only reporting endpoint=] to the item's
                 [=embedder policy/report only reporting endpoint=].
+4.  Otherwise:
+
+    1.   Set |policy| to |response policy|.
+
+5.  Set |global|'s [=WorkerGlobalScope/embedder policy=] to |policy|.
+
+### Checking a global object's Embedder policy ### {#check-embedder-policy-for-global}
+<div algorithm="to check a global object's embedder policy">
+To <dfn abstract-op lt="queue coep worker violation">Queue a Cross-Origin Embedder Policy
+violation on worker initialization</dfn> given a [=request=] (|request|), a string (|endpoint|) and an
+[=environment settings object=] (|settings|), run the following steps:
+
+1.  Let |blocked url| be |request|'s [=request/URL=].
+
+    Note: This is not |request|'s [=request/current URL=] in order to avoid redirect
+    information leak.
```

Comment:
```suggestion
    Note: This is not |request|'s [=request/current URL=] in order to avoid leaking information
    about redirect targets (see https://w3c.github.io/webappsec-csp/#security-violation-reports).
```